app/templates/components/filter-by.hbs

```diff
@@ -1,10 +1,61 @@
-{{#if isShowingBody}}
-  <div class="filter-table">
-    <ul>
-      <li>Teams</li>
+{{!-- Filter By --}}
+
+<div class="filter">
+  <BsDropdown as |dd|>
+    <dd.toggle class="filter-by"><span>Filter by</span><p
+        class="emp"
+        {{action "filterby"}}
+      >{{if currentTeam currentTeam "All Employees"}}</p>
+      <img
+        src="/assets/images/accordion-open.svg"
+        alt="img"
+        class="open {{if clicked 'turn'}}"
+      />
+    </dd.toggle>
+    <dd.menu>
+      <li class="team">Teams</li>
       {{#each myTeamMembers as |myTeam|}}
-        <li {{action "isFilter" myTeam.team}}>{{myTeam.team}}</li>
+        <li
+          class="filter-table"
+          {{action "isFilter" myTeam.team}}
+        >{{myTeam.team}}</li>
       {{/each}}
-    </ul>
-  </div>
-{{/if}}
\ No newline at end of file
+    </dd.menu>
+  </BsDropdown>
+</div>
+
+{{!-- Sort By --}}
+
+<div class="sort">
+  <BsDropdown as |dd|>
+    <dd.toggle class="filter-by"><span>Sort by</span><p
+        class="emp"
+        {{action "sortby"}}
+      >{{if sortDisplay sortDisplay "First Name"}}</p>
+      <img
+        src="/assets/images/accordion-open.svg"
+        alt="img"
+        class="open {{if sortByclicked 'turn'}}"
+      />
+    </dd.toggle>
+    <dd.menu>
+      <div class="menu">
+        {{#each sortArray as |sortType|}}
+          <li
+            class="sort-table
+              {{if (eq this.selectedValue sortType.name) 'tick'}}"
+            {{action "sortFunc" sortType.name sortType.displayName}}
+          >{{sortType.displayName}}</li>
+        {{/each}}
+        <div class="dropdown-divider divider"></div>
+        {{#each sortOrderArray as |sortOrderType|}}
+          <li
+            class="sort-method
```

Comment:
Use the button inside the li tag and add action to the button

Reply:
I have fixed this change.